src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int INDEX_OF_DATE = 0;
+    private static final int INDEX_OF_NAME = 1;
+    private static final int INDEX_OF_HOURS = 2;
+    private static final int INDEX_OF_RATE = 3;
+    private static final String BEGIN_OF_REPORT = "Report for period ";
+    private static final String DASH_FOR_REPORT = " - ";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the method implementation.